scripts/make_release.sh

```diff
@@ -0,0 +1,4 @@
+./scripts/uninstall.sh
+./scripts/clean_up.sh
+python setup.py register
+python setup.py sdist bdist_wheel upload
```

Comment:
I would strongly suggest you to take a look at zest.releaser.

My usual workflow to make a release with it is:

```
source bin/activate
pip install -U zest.releaser[recommended]
fullrelease
```

zest.relaser takes care of all the boring bits and on top of that uploads the files with twine, so your pypi credentials are not sent in clear text.